src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -141,17 +143,26 @@ public static void checkWarningsAndErrors(SscsCaseData sscsCaseData, ScannedDocu
         }
 
         String scannedDocumentType = scannedDocument.getValue().getType();
-        String actionCode = sscsCaseData.getFurtherEvidenceAction().getValue().getCode();
 
         if (ScannedDocumentType.SET_ASIDE_APPLICATION.getValue().equals(scannedDocumentType)
-                && !SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getCode().equals(actionCode)) {
+            && !SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getCode().equals(actionCode)
+        ) {
+            preSubmitCallbackResponse.addError(String
+                .format("'Further Evidence Action' must be set to '%s'",
+                    SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getLabel()));
+        }
+
+        if (ScannedDocumentType.CORRECTION_APPLICATION.getValue().equals(scannedDocumentType)
+            && !SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getCode().equals(actionCode)
+            && !ADMIN_ACTION_CORRECTION.getCode().equals(actionCode)
+        ) {
```

Comment:
```suggestion
            boolean IsDocTypeCorrectionApplication = ScannedDocumentType.CORRECTION_APPLICATION.getValue().equals(scannedDocumentType)
            boolean IsNotInterlocReviewbyJudge =  !SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getCode().equals(actionCode)
            boolean isNotAdminActionCorrection = !ADMIN_ACTION_CORRECTION.getCode().equals(actionCode)
            
            if (IsDocTypeCorrectionApplication && IsNotInterlocReviewbyJudge && isNotAdminActionCorrection) 
```
Additionally that whole IF condition could be putted to method that returns bool

Reply:
done